docs/intro-tutorial.rst

```diff
@@ -1,23 +1,32 @@
 Introduction to Mesa - Tutorial
-================================
+*********************************
```

Comment:
I really hate RST so much. I kind of want to abandon it and just go to .md files for the docs. I feel like it would be so much easier.